tools/hsettp/hsettp.c

```diff
@@ -1237,7 +1237,7 @@ operation_usage(FILE * const output, cJSON * const path, cJSON * const method)
     fprintf(output, "\nColumns:\n ");
     for (size_t i = 0; i < columnc; i++)
         fprintf(output, " %s", columnv[i]);
-    fprintf(output, "\n\n");
+    fprintf(output, "\n");
```

Comment:
I prefer it, helps separate the output from the next prompt...

Reply:
just make your PS1 start with a newline :^)

Reply:
I can add it back, but the inconsistency :/